### PR TITLE
Calvin/add autocomplete

### PR DESCRIPTION
**Pull request by calvind777**
@eric-ls i had to change the css to get the google map to show up, and i can't find a way around specifying the exact size in px. but you probably can cuz u r frontend master. otherwise, everything should work correctly!

### CODE REVIEW

File: app/assets/javascripts/components/dashboard/create_location.jsx [hunk 1]
```diff
@@ -32,8 +33,64 @@ class LocationCreationForm extends DefaultForm {
     this.setState({ showModal: true });
   }
 
+  initMap = (e) => {
+        
+        var locationForm = this;
+        console.log(e);
+        console.log(this.state.showModal);
```
Comment (eric-ls) [inline app/assets/javascripts/components/dashboard/create_location.jsx]: remember to remove logs!

File: app/assets/javascripts/components/dashboard/create_location.jsx [hunk 2]
```diff
@@ -32,8 +33,64 @@ class LocationCreationForm extends DefaultForm {
...
+        if (document.getElementById('map').innerHTML||!e) {  
+          return;
+        }
+        var map = new google.maps.Map(this.mapDiv, {
+          center: {lat:37.791569, lng:-122.389938},
```
Comment (eric-ls) [inline app/assets/javascripts/components/dashboard/create_location.jsx]: is there a way to center at the current location? if not, this is fine for now

File: app/assets/javascripts/components/dashboard/create_location.jsx [hunk 3]
```diff
@@ -32,8 +33,64 @@ class LocationCreationForm extends DefaultForm {
...
+          zoom: 8
+        });
+        var marker = new google.maps.Marker({
+          position: {lat:37.791569, lng:-122.389938},
+          map: map
+        });
+
+        //google.maps.event.trigger(map, 'resize');
```
Comment (eric-ls) [inline app/assets/javascripts/components/dashboard/create_location.jsx]: remove commented lines

**Comment by TheXienator** [comment]
new pull request replaced this one

**Pull request closed without merging**